package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "mern-boilerplate",
```

Comment:
should this file be in the backend folder? is there a README explaining the technical design and instructions to run backend + frontend?

Reply:
Package.json files have been updated. Now reads "Proof-of-attendance-backend"